bento_beacon/utils/katsu_utils.py

```diff
@@ -141,21 +141,30 @@ def get_katsu_config_search_fields():
 }
 
 
-# assume json query already validated
 # convert an individual beacon filter into bento format
 def bento_query_expression(q):
     # break up phenopackets property name with "#resolve" appended at the front
     katsu_key = ["#resolve", *q["id"].split(".")]
 
-    # extra handling for negation, "!" is always negated equality
+    beacon_value = q["value"]
+    has_wildcard = "%" in beacon_value
+
+    # reject meaningless cases
+    if has_wildcard and q["operator"] in ["<", "<=", ">", ">="]:
```

Comment:
i prefer tuples for these inline lists, since tuples are immutable. alternatively, this could be given a constant name like `INEQUALITY_OPERATORS` and made a `frozenset`

Reply:
changed to tuple, thanks